packages/node/src/app/analytics-node.ts

```diff
@@ -51,6 +52,7 @@ export class Analytics extends NodeEmitter implements CoreAnalytics {
         httpRequestTimeout: settings.httpRequestTimeout,
         disable: settings.disable,
         flushInterval,
+        customclient: settings.customClient ?? new DefaultFetchClient(),
```

Comment:
Thoughts on calling this setting "httpClient" instead of customClient? 

- pretty idiomatic
- Communicates that this is specifically an http client  
- Let's us add an option for more general non-http clients if we ever need to be (which would override hyypClient)

Reply:
Yeah, I think it's better, changing